com.gitlab.inendi.Inspector.yaml

```
@@ -0,0 +1,23 @@
+app-id: com.gitlab.inendi.Inspector
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
```

```suggestion
runtime-version: '21.08'
```

20.08 will be EOL in 3 month.

---

Yes, I already started to work on the migration to runtime 21.08 but there is still ongoing work.